File: output/gemini-3.1-pro-preview/zero/cpp/Relógio.cpp

#include <iostream>

using namespace std;

int main() {
    ios_base::sync_with_stdio(false);
    cin.tie(NULL);

    long long h, m, s, t;
    if (cin >> h >> m >> s >> t) {
        long long total_seconds = h * 3600 + m * 60 + s + t;
        total_seconds %= 86400;

        long long new_h = total_seconds / 3600;
        long long new_m = (total_seconds % 3600) / 60;
        long long new_s = total_seconds % 60;

        cout << new_h << "\n" << new_m << "\n" << new_s << "\n";
    }

    return 0;
}

File: output/gemini-3.1-pro-preview/zero/python/Relógio.py
import sys

def main():
    input_data = sys.stdin.read().split()
    if not input_data:
        return
    
    H = int(input_data[0])
    M = int(input_data[1])
    S = int(input_data[2])
    T = int(input_data[3])
    
    total_seconds = H * 3600 + M * 60 + S + T
    
    new_S = total_seconds % 60
    total_minutes = total_seconds // 60
    new_M = total_minutes % 60
    total_hours = total_minutes // 60
    new_H = total_hours % 24
    
    print(new_H)
    print(new_M)
    print(new_S)

if __name__ == '__main__':
    main()